README.md

```diff
@@ -7,11 +7,81 @@ create sets, packages and images for the OPNsense project.
 Setting up a build system
 =========================
 
-Install [FreeBSD](https://www.freebsd.org/) 13.0-RELEASE for amd64
+Install [FreeBSD](https://www.freebsd.org/) 13.1-RELEASE for amd64
 on a machine with at least 25GB of hard disk (UFS works better than ZFS)
 and at least 4GB of RAM to successfully build all standard images.
-All tasks require a root user.  Do the following to grab the repositories
-(overwriting standard ports and src):
+All tasks require a root user.
+
+
+*pkg* Shenanigans
+-----------------
+
+Upstream keeps making incompatible changes to ``pkg`` which causes build
+failures.  In order to work around this problem you must use the OPNsense
+version of pkg, not the FreeBSD version of pkg.  This will require some
+non-standard setup to accomplish.
+
+This is necessary because OPNsense builds within a jail and most but not
+all operations happen with the jails version of `pkg`. There are some aspects
+of the build process that operate outside the jail and those steps require
+interoperability between the base pkg and the jail pkg.
+
+To enable this compatibility you will need to use the OPNsense package
+repositories instead of the FreeBSD ones.
+
+First install git and acquire the `core` repository.  Please note that this
```

Comment:
I'm a little doubting if we have to make all of these instructions vs. simply adding a "fix" make target that runs the required sequence and can be easily tested and leaves almost no room for error? :)

Reply:
I've created a 'make fix' target that should be version independent and with with the variables from the Makefile.